test/TypeScriptEngine.spec.ts

```diff
@@ -190,7 +190,7 @@ describe("TypeScriptEngine tests", () => {
         const result: InterfaceDeclaration = engine.createInterface(baseInterfaceDecl);
         expect(result).to.deep.equal(ts.createInterfaceDeclaration(
             /* decorators */ undefined,
-            /* modifiers */ undefined,
+            /* modifiers */ [ts.createModifier(SyntaxKind.ExportKeyword)],
```

Comment:
I guess we don't need `default` here since our DSL only supports one interface/class per `.ts` file?

Reply:
Yes but also this doesn't produce ```default```